### ISSUE TITLE
Bug Winbox in Ubuntu 24.04 LTS

### ISSUE THREAD

**Issue by falbuque** [panaceya/winbox#51]
Hi, a problem for winbox in ubuntu 24.04 LTS.
Instalation is ok, more when i open, the program starts wine, but does not open winbox,

**Comment by RivoHenfri** [comment]
I got the same issue
000d:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
000d:err:winediag:nodrv_CreateWindow The explorer process failed to start.
wine-9.0

**Comment by mfth12** [comment]
yes, me too. it can't start
![image](https://github.com/panaceya/winbox/assets/53788525/0abfc3f0-9c8d-4575-9a75-40471b177fdd)

**Comment by cspencer51** [comment]
Having this same issue.

**Describe the bug**
The application fails to start.
```
$ winbox 
000b:err:module:__wine_process_init failed to load L"C:\\windows\\system32\\wineboot.exe", error c0000018
0009:err:environ:run_wineboot failed to start wineboot c0000018
 $ 000d:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
000d:err:winediag:nodrv_CreateWindow The explorer process failed to start.
```

I also see these messages from AppArmor. I don't see these on Ubuntu 22.04:
```
[73778.766915] audit: type=1400 audit(1715194447.032:710): apparmor="DENIED" operation="open" class="file" profile="snap.winbox.winbox" name="/run/" pid=134765 comm="wineserver" requested_mask="r" denied_mask="r" fsuid=1002 ouid=0
[73778.853985] audit: type=1400 audit(1715194447.119:712): apparmor="DENIED" operation="connect" class="file" profile="snap.winbox.winbox" name="/run/cups/cups.sock" pid=134772 comm="winbox64.exe" requested_mask="wr" denied_mask="wr" fsuid=1002 ouid=0
```

**To Reproduce**
Steps to reproduce the behavior:
1. Run `winbox` from the command line _or_ launch the Winbox snap application.

**Expected behavior**
The application should start.

**Screenshots**
N/A

**Distribution (please complete the following information):**

Name: ubuntu
Version: 24.04
Architecture: amd64

** snap version **
```
snap    2.62+24.04build1
snapd   2.62+24.04build1
series  16
ubuntu  24.04
kernel  6.8.0-31-generic
```
** snap info winbox **
```
name:      winbox
summary:   small utility that allows administration of MikroTik RouterOS
publisher: ThePanaceya
store-url: https://snapcraft.io/winbox
license:   unset
description: |
  Winbox is a small utility that allows administration of MikroTik RouterOS using a fast and simple
  GUI.
  
  Snap conatins Wine runtime and winbox client.
  
  Readme: https://github.com/panaceya/winbox/blob/main/README.md
commands:
  - winbox
snap-id:      UdK6K4ZjRfjL0wRFh0tWGO7vPhA4Qge4
tracking:     latest/stable
refresh-date: yesterday at 15:12 PDT
channels:
  latest/stable:    3.40 2024-03-09 (135) 47MB -
  latest/candidate: ↑                          
  latest/beta:      ↑                          
  latest/edge:      3.40 2024-03-05 (135) 47MB -
installed:          3.40            (135) 47MB -
```
** snap list  winbox **
```
Name    Version  Rev  Tracking       Publisher    Notes
winbox  3.40     135  latest/stable  thepanaceya  -
```

**Comment by panaceya** [comment]
Fixed in #53. 
You need before refresh by `snap refresh winbox --stable`.
